services/spar/package.yaml

```diff
@@ -90,6 +128,8 @@ tests:
       - spar
       - uri-bytestring
 
+    default-extensions: *extension-list
```

Comment:
I think you don't need that, just specifying default-extensions at the top level is enough (same as with `dependencies`).